spyderlib/spyder.py

```diff
@@ -2646,6 +2646,9 @@ def start_open_files_server(self):
                 # To avoid a traceback after closing on Windows
                 if e.args[0] == eintr:
                     continue
+                # handle a connection abort on close error
+                if e.args[0] in [errno.ECONNABORTED, errno.ENOTSOCK]:
+                    return                

```

Comment:
I also forgot to mention that `errno.ENOTSOCK` is not available on Windows. See my referenced commit for the right error on Win.